src/alwaysAck.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+/* @flow */
+
+import type { MessageHandler } from './index'
+
+import { ackOnComplete, ackOnError } from './ackOn'
+import parseAndHandleMessage from './parseAndHandleMessage'
+
+export default (handleError:(e:mixed) => Promise<mixed>, f:(message:Object) => Promise<mixed>): MessageHandler =>
+  parseAndHandleMessage(x => x, ackOnError(handleError), ackOnComplete, f)
```

Comment:
Hmmm, I don't think either is strictly more general.  For example, `alwaysAck` can't nack, whereas `parseAndHandleMessage` can.  I bet there's some other yet-to-be-written function that is more general than either!